website/docs/api/attributes.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Attributes
+teaser: Token attributes
+source: spacy/attrs.pyx
+---
+
+[Token](/api/token) attributes are specified using internal IDs in many places
+including:
+
+- [`Matcher` patterns](/api/matcher#patterns),
+- [`Doc.to_array`](/api/doc#to_array) and
+  [`Doc.from_array`](/api/doc#from_array)
+- [`Doc.has_annotation`](/api/doc#has_annotation)
+- [`MultiHashEmbed`](/api/architectures#MultiHashEmbed) Tok2Vec architecture
+  `attrs`
+
+> ```python
+> import spacy
+> from spacy.attrs import DEP
+>
+> nlp = spacy.blank("en")
+> doc = nlp("There are many attributes.")
+>
+> # DEP always has the same internal value
+> assert DEP == 76
+>
+> # "DEP" is automatically converted to DEP
+> assert DEP == nlp.vocab.strings["DEP"]
+> assert doc.has_annotation(DEP) == doc.has_annotation("DEP")
+>
+> # look up IDs in spacy.attrs.IDS
+> from spacy.attrs import IDS
+> assert IDS["DEP"] == DEP
+> ```
+
+All methods automatically convert between the string version of an ID (`"DEP"`)
+and the internal integer symbols (`DEP`). The internal IDs can be imported from
+`spacy.attrs` or retrieved from the [`StringStore`](/api/stringstore).
+
+The corresponding [`Token` attributes](/api/token#attributes) can be accessed
+using the attribute name in lowercase, e.g. `token.orth` or `token.length`.
+
+String attributes are stored internally as integer hash values from the
+[`StringStore`](/api/stringstore). The integer value is accessed as `Token.attr`
+e.g. `token.dep` while the string value is accessed by appending `_` as in
+`token.dep_`.
```

Comment:
There's something off with the flow of this paragraph to me, like it should be incorporated in the previous 2 paragraphs somehow? Mentioning the `StringStore` twice in slightly different context is a bit confusing, and the same is true for explaining how to access `token.attr` twice (kind of)